addon/components/frost-bunsen-cell/component.js

```diff
@@ -1,12 +1,11 @@
 import _ from 'lodash'
-import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 
-import PropTypesMixin, {PropTypes} from 'ember-frost-bunsen/mixins/prop-types'
+import FrostComponent, {PropTypes} from 'ember-frost-component'
```

Comment:
I guess my main argument for FrostComponent would be the ability to have multiple mixins without having to do an import for each (e.g. proptypes, ember-computed-decorators, etc.)